src/App.js

```diff
@@ -1,13 +1,29 @@
 import "./App.css";
 import Header from "./components/Header";
-import Body from "./components/Body";
 import Footer from "./components/Footer";
+import Reviews from "./components/Reviews";
+import Review from "./components/Review";
+import { Routes, Route } from "react-router-dom";
+import { useState } from "react";
 
 function App() {
+  const [isLoading, setIsLoading] = useState(true);
+
   return (
     <div className="App">
       <Header />
-      <Body />
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <Reviews isLoading={isLoading} setIsLoading={setIsLoading} />
+          }
+        />
+        <Route
+          path="/:review_id"
```

Comment:
this path is rather meaningless in practice, so use something like `reviews/:review_id`